### PR TITLE
Ffmpeg libraries

### PR DESCRIPTION
**Pull request by slycordinator**
added links to descriptions of which libraries are included for ffmpeg packages

## Description
For the ffmpeg and ffmpeg-full packages, in the readme, I added links to the site that shows the libraries included for each

## Motivation and Context
* Currently, packages refer to an "essentials" and "full" build, but in the upstream project there is no such thing as either of those. Long-time users of ffmpeg could/do look at it wondering what either of them mean.
* fixes https://github.com/chocolatey-community/chocolatey-packages/issues/1765

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Documentation change
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [X] My change requires a change to documentation (this usually means the notes in the description of a package).
- [X] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [X] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- ~[ ] The added/modified package passed install/uninstall in the chocolatey test environment.~
- [ ] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/ffmpeg-full/README.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ Security is a high priority and code review is always done with security in mind
 ## Notes
 
 * This package is for the stable static full ffmpeg version.
-
+* For a list of libraries included, go to https://www.gyan.dev/ffmpeg/builds/#libraries
```
Comment (AdmiringWorm) [inline automatic/ffmpeg-full/README.md:13]: Just out of curiosity, should we have a mention here that the libraries listed on that page may only be valid for the latest version of the package?

Thoughts @TheCakeIsNaOH @gep13

File: automatic/ffmpeg-full/README.md [hunk 2]
```diff
@@ -10,4 +10,5 @@ Security is a high priority and code review is always done with security in mind
...
+* (provides everything included in the 'essentials' build with additional libraries)
```
Comment (AdmiringWorm) [inline automatic/ffmpeg-full/README.md:14]: This item will show up as its own list item.

instead, remove the star, and have the text start at the same column as the preceding line.

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]

File: automatic/ffmpeg-full/README.md (hunk 1 above)
Comment (TheCakeIsNaOH) [reply automatic/ffmpeg-full/README.md:13]: Yup, agreed that it should note that the list of libraries is for the current latest release.

File: automatic/ffmpeg-full/README.md (hunk 2 above)
Comment (slycordinator) [reply automatic/ffmpeg-full/README.md]: I added that change to the branch for the PR

File: automatic/ffmpeg-full/README.md (hunk 1 above)
Comment (slycordinator) [reply automatic/ffmpeg-full/README.md]: I added a note for the fact that the list may only apply to the lastest version

It's now in the same branch as I used for this PR.

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Comment by AdmiringWorm** [comment]
@slycordinator your changes have been merged, thanks for your contribution 👍

The documentation changes will be part of any future package versions of the ffmpeg packages that will be pushed to the Chocolatey Community Repository.

**Comment by GyanD** [comment]
The list of external libraries in a build is already present in README.txt inside the package.

Starting with the next release, the README.txt will contain a list of all components, whether internal or external.

**Pull request merged**